docs/design.md

```diff
@@ -30,6 +30,15 @@ parameters:
     type: int
     default: 3306
 ```
+Or with an empty parameters field
+```yaml
+name: fusor/etherpad-apb
+description: Note taking web application
+bindable: true
+async: optional
+parameters: []
```

Comment:
Actually, the defaults for these values are only known by this file.  What ends up happening if you don't pass any params?